src/modules/bytes.ts

```diff
@@ -44,6 +44,7 @@ export async function upload(
   return {
     reference: response.data.reference,
     tagUid: response.headers['swarm-tag'] ? makeTagUid(response.headers['swarm-tag']) : undefined,
+    history_address: response.headers['swarm-act-history-address'] || '',
```

Comment:
extend the other endpoints with act history header: soc, chunk

Reply:
extended: https://github.com/Solar-Punk-Ltd/bee-js/commit/5c9cf24459db299a21520a8dd1c6c96927253dea